### DISCUSSION TITLE
bitbucket integration stopped working

### DISCUSSION THREAD

**Discussion by sokol8** [vercel/vercel#4633] [Help]
I was trying to overcome a limitation of bitbucket API and removed my Vercel-Bitbucket integration.

1) After I tried to add bitbucket repo again I faced an error 
`Error:The bitbucket repository can't be retrieved.`
The system doesn't ask for my bitbucket account or anything.

2) When I try to add a github repo I get into a "Connect your Github account" step

3) To remove suspicion from my bitbucket account, I created a new repo under another btibucket account of mine and tried to import it but the process failed with the same error
`Error:The bitbucket repository can't be retrieved.`

I suspect that there's either
- something happened with my Vercel account or
- there's a broader issue Vercel-Bitbucket integration

Has anyone experienced something similar recently?
How did you solve that?

**Comment by sokol8** [comment]
I cancelled my Pro team trial. 
After cancellation there appeared "Connect your git account" screen inside "Git Integrations" on my Hobby plan.
I tried to connect bitbucket but stopped the process half-way
When I returned to "Git Integration" page inside Settings under my Hobby plan, I see a status list of Github, GitLab, Bitbucket Integrations (which all show disconnected status) and no way to connect my bitbucket account (or any other account)

**Comment by paulogdm** [comment]
Please reach out [to support](https://vercel.com/support/request) so we can take a look at the issue as soon as possible.

**Comment by sokol8** [comment]
Hi @paulogdm 
I contacted Vercel support regarding this issue yesterday - Jun 11, 2020, 12:14 PM Eastern time

"Thank you for contacting Vercel Support. Your message has been received and is being reviewed by the Vercel Support team."

No reply so far

**Reply by paulogdm** [reply]
We will get to your ticket soon. The "free plan" queue is quite busy atm, unfortunately.

**Comment by sokol8** [comment]
Hi Paul,

I totally understand that free vs paid plan trial. I just want to point out
that I was at the Pro plan trial and was about to start paying you
$40/month within a week from incident.

Unfortunately, such blocking issues moved me out to Netlify quickly. I
simply cannot block the work of the team and have no reply from support for
days.
Hope to try Vercel on another project.

Please observe where this whole issue came from:
- Bitbucket has granular access per repo so I can give access to a
particular developer to a particular repo.
- If I add my bitbucket to Vercel - a developer (not Admin) suddenly has
access to all my Bitbcuket repos. That's a huge security problem.





On Sat, Jun 13, 2020 at 11:09 AM paulogdm <notifications@github.com> wrote:

> We will get to your ticket soon. The "free plan" queue is quite busy atm,
> unfortunately.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vercel/vercel/discussions/4633#discussioncomment-25864>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAH4GE4T4PUHAWCDZKUFV2TRWOJDJANCNFSM4N32D6FQ>
> .
>


-- 
Best regards,

Kostiantyn Sokolinskyi
*sokol8@gmail.com <http://gmail.com/>*

**Reply by paulogdm** [reply]
Thanks for letting me know. I have relayed your feedback internally.

**Comment by sokol8** [comment]
Hope that will help streamline your support / QA process.

I understand that you guys are growing crazy now and this takes its toll. I
really enjoyed the platform and hope to try it again soon. Yet I cannot
block my business.

On Wed, Jun 24, 2020 at 09:39 paulogdm <notifications@github.com> wrote:

> Thanks for letting me know. I have relayed your feedback internally.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vercel/vercel/discussions/4633#discussioncomment-29852>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAH4GE6VN4XBH6U75BMSNRDRYH6ZLANCNFSM4N32D6FQ>
> .
>
-- 
Best regards,

Kostiantyn Sokolinskyi
*sokol8@gmail.com <http://gmail.com/>*
